src/controller/python/test/test_scripts/commissioning_test.py

```diff
@@ -88,20 +115,28 @@ def main():
         nodeid=112233, paaTrustStorePath=options.paaTrustStorePath, testCommissioner=True)
 
     logger.info("Testing discovery")
-    FailIfNot(test.TestDiscovery(discriminator=TEST_DISCRIMINATOR),
+    FailIfNot(test.TestDiscovery(discriminator=options.discriminator),
               "Failed to discover any devices.")
 
     FailIfNot(test.SetNetworkCommissioningParameters(dataset=TEST_THREAD_NETWORK_DATASET_TLV),
               "Failed to finish network commissioning")
 
-    logger.info("Testing key exchange")
-    FailIfNot(test.TestKeyExchange(ip=options.deviceAddress,
-                                   setuppin=20202021,
-                                   nodeid=1),
-              "Failed to finish key exchange")
+    if options.deviceAddress:
+        logger.info("Testing key exchange (IP)")
```

Comment:
Why "key exchange" ? It's not key exchange, it's full commissioning

Reply:
Renamed key exchange -> commissioning